doc/sphinx/practical-tools/utilities.rst

```diff
@@ -499,6 +499,40 @@ To build, say, two targets foo.vo and bar.vo in parallel one can use
     (``.PHONY`` or not) please use ``CoqMakefile.local``.
 
 
+Precompiling for ``native_compute``
++++++++++++++++++++++++++++++++++++
+
+To compile files for ``native_compute``, one can use the
+``-native-compiler yes`` option of |Coq|, for instance by putting the
+following in a :ref:`coqmakefilelocal` file:
+
+::
+
+    COQEXTRAFLAGS := -native-compiler yes
+
+The generated ``CoqMakefile`` installation target will then take care
+of installing the extra ``.coq-native`` directories.
+
+.. note::
+
+   As an alternative to modifying any file, one can set the ``COQC``
+   environment variable when calling ``coq_makefile``:
+
+   ::
+
+      COQC="$(which coqc) -native-compiler yes" coq_makefile -f _CoqProject -o CoqMakefile
```

Comment:
I don't think we want to recommend this, especially if we end up quoting COQC at the use point (if someone installs it in a directory with spaces like `Program Files` we really should quote it).
What's wrong with modifying COQEXTRAFLAGS? Unlike what the above implies modifying it as an environment variable should work fine too.

Reply:
Nothing by itself. The advantage I see to COQC is that it also works with libraries built with Remakefile (Flocq and CoqInterval for instance). I checked in coq_makefile generated Makefiles, the COQC variable is not quoted when it is provided by the user (only when it is not) so that's does not seem to be an issue.

Reply:
Please don't suggest this, there is a dedicated variable.

If you want to cover remake in the doc, do so.

Reply:
Ok, done so.